RedBugs/src/main/java/Data/UserAdder.java

```diff
@@ -1,43 +1,109 @@
 package Data;
 
+import java.util.InputMismatchException;
 import java.util.Scanner;
 
 public class UserAdder {
     public static User createUser() {
 
+        User user = new User();
+        int id = 0;
+
+        String anotherUser = null;
         Scanner scanner = new Scanner(System.in);
 
-        User user = new User();
+        do {
+
+            System.out.println("Rozpoczynasz procedurę wprowadzania nowego użytkownika. ");
+
+            try {
+
+                System.out.println("Wprowadź imię użytkownika:");
+
+                String name = scanner.nextLine().toLowerCase();
+                user.setName(name);
+
+                System.out.println("Wprowadź nazwisko użytkownika:");
 
-        System.out.println("Wprowadź imię użytkownika:");
+                String surname = scanner.nextLine().toLowerCase();
+                user.setSurname(surname);
 
-        String name = scanner.nextLine();
-        user.setName(name);
+                System.out.println("Wprowadź login użytkownika:");
 
-        System.out.println("Wprowadź nazwisko użytkownika:");
+                String login = scanner.nextLine().toLowerCase();
+                user.setLogin(login);
 
-        String surname = scanner.nextLine();
-        user.setSurname(surname);
+                System.out.println("Wprowadź hasło użytkownika:");
 
-        System.out.println("Wprowadź login użytkownika:");
+                String password = scanner.nextLine().toLowerCase();
+                user.setPassword(password);
 
-        String login = scanner.nextLine();
-        user.setLogin(login);
+                user.setId(id + 1);
 
-        System.out.println("Wprowadź hasło użytkownika:");
+                System.out.println("Czy chcesz stworzyć kolejnego użytkownika? Jeżeli tak, wciśnij 'y' ");
 
-        String password = scanner.nextLine();
-        user.setPassword(password);
+                anotherUser = scanner.nextLine();
 
-        // ID should be generated automiaticly - work in progress
-        System.out.println("Wprowadź ID użytkownika:");
+                System.out.println("Zakończono procedurę tworzenia uzytkownika");
 
-        int id = scanner.nextInt();
-        user.setId(id);
+            } catch (InputMismatchException e) {
+
+                System.out.println("Podana przez Ciebie wartość nie jest prawidłowa. ");
+
+            }
+        } while (anotherUser.equals("y"));
 
         return user;
     }
 
+    public static void logUser() {
+
+        Scanner scanner = new Scanner(System.in);
+
+        User user = createUser();
+        //temporarly values for program checkout
+        String iPassword = user.getPassword();
+        String iLogin = user.getLogin();
+        int iid = user.getId();
+
+
+        //test
+        System.out.println(user.getPassword());
+        System.out.println(user.getLogin());
+
+        System.out.println("Rozpocząłeś procedurę logowania.");
+        System.out.printf("Podaj login użytkownika:");
```

Comment:
```suggestion
        System.out.println("Logowanie");
        System.out.printf("login:");
```
Myślę, że tak wystarczy.